DemoServer/AdditionalLanguages/nodejs/demo/queries/pagingQueryResults/metadata.json

```
@@ -0,0 +1,227 @@
+                    "Title": "Querying: Distinct Paging",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/docs/article-page/4.2/nodejs/indexes/querying/distinct#paging"
+                },
+                //TODO
+                //{
+                //    "Title": "Fanout indexes Paging",
+                //    "Type" : "link",
+                //    "Url"  : "https://ravendb.net/docs/article-page/4.2/nodejs/indexes/fanout-indexes#paging"
+                //},
+                {
+                    "Title": "Readme - skip()",
+                    "Type" : "link",
+                    "Url"  : "https://github.com/ravendb/ravendb-nodejs-client#skip"
+                }
+            ]
+        },
+        {
+            "Title": "Take results",
+            "Slug": "step-4",
+            "DescriptionLines": [
+                "* Use `take()` to specify the number of results to actually take (return) from the total available result set.",
+                "",
+                "* The position from where to start 'taking' is set by the previous 'skip()' method.",
+                "",
+                "* If no results were skipped - when Not using skip() at all, or when using skip(0) - then the first 'resultsToTake' documents will be returned from the result set."
+            ],
+            "Assets": [
+                {
+                    "Title": "Querying: Paging",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/docs/article-page/4.2/nodejs/indexes/querying/paging"
+                },
+                {
+                    "Title": "Readme - take()",
+                    "Type" : "link",
+                    "Url"  : "https://github.com/ravendb/ravendb-nodejs-client#take"
+                }
+            ]
+        },
+        {
+            "Title": "Execute the query",
+            "Slug": "step-5",
+            "DescriptionLines": [
+                "* Execute the query by calling `all()`.",
+                "",
+                "* _'pagedResults'_ will contain only 'resultsToTake' documents starting from the 'resultsToSkip' location from the result set.",
+                "",
+                "* The output entities returned by the query are tracked by the Session as usual.<br>"
+            ],
+            "Assets": [
+                {
+                    "Title": "Execute the Query - Query Overview Demo",
+                    "Type" : "demo",
+                    "Url"  : "/nodejs/queries/query-overview#step-3"
+                }
+            ]
+        },
+        {
+            "Title": "Get total number of results",
+            "Slug": "step-6",
+            "DescriptionLines": [
+                "* When retrieving only 'page size' results, you might still need to know the total number of results.",
+                "",
+                "* The total number of documents in the result set can be retrieved from: `stats.totalResults()`.<br>",
```

Are you sure about that parens there?
```suggestion
                "* The total number of documents in the result set can be retrieved from: `stats.totalResults`.<br>",
```

---

done - removed